.github/workflows/stale-prs.yml

```diff
@@ -0,0 +1,113 @@
+name: Mark stale pull requests in Shepherd repo
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
```

Comment:
I would suggest adding a timeout to avoid an endless running script in the case something unexpected goes wrong.